### PR TITLE
Added File Storage and Manipulation category

### PR DESCRIPTION
**Pull request by 007Cat24**
Included Dropbox, Amazon S3, Cloudinary and Filestack.

### CODE REVIEW

**Comment by 007Cat24** [comment]
@abhishekbanthia Here we go

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ This is an attempt to categorise different APIs scoured from the web which make
 - [Data Science] (#data-science)
 - [Dictionary] (#dictionary)
 - [Fitness] (#fitness)
+- [File Storage and Manipulation] (#file-storage-and-manipulation)
```
Comment (abhishekbanthia) [inline README.md]: Can you move this above the Fitness category?

Comment (007Cat24) [inline README.md]: Sure

**Comment by 007Cat24** [comment]
Done

**Pull request merged**